scripts/createEDM4hepFile.py

```diff
@@ -6,51 +6,14 @@
 import podio
 import edm4hep
 from itertools import count
-import argparse
-import sys
-
-frames = 3  # How many frames or events will be written
-vectorsize = 5  # For vector members, each vector member will have this size
-counter = count()  # next(counter) will return 0, 1, 2, ...
-
-parser = argparse.ArgumentParser(description="Create a file with EDM4hep data")
-parser.add_argument(
-    "--rntuple", action="store_true", help="Use a ROOT ntuple instead of EDM4hep"
-)
-parser.add_argument(
-    "--output-file", type=str, help="Output file name", default="edm4hep.root"
-)
-args = parser.parse_args()
-output_file = args.output_file
-
-if args.rntuple:
-    try:
-        writer = podio.root_io.RNTupleWriter(output_file)
-    except AttributeError:
-        print("The RNTuple writer from podio is not available but was requested")
-        sys.exit(1)
-else:
-    writer = podio.root_io.Writer(output_file)
-
-for i in range(frames):
-    print(f"Writing frame {i}")
-    frame = podio.Frame()
 
-    # Make covariance matrices needed later
+FRAMES = 3  # How many frames or events will be written
+VECTORSIZE = 5  # For vector members, each vector member will have this size
 
-    # With the current version of cppyy (from ROOT 6.30.06)
-    # it's not possible to initialize an std::array from a list
-    # In future versions (works with 6.32.02) it will be possible
-    cov3f = edm4hep.CovMatrix3f()
-    for i in range(6):
-        cov3f[i] = next(counter)
-    cov4f = edm4hep.CovMatrix4f()
-    for i in range(10):
-        cov4f[i] = next(counter)
-    cov6f = edm4hep.CovMatrix6f()
-    for i in range(21):
-        cov6f[i] = next(counter)
 
+def create_EventHeaderCollection(vectorsize):
+    """Create an EventHeaderCollection"""
+    counter = count()
```

Comment:
Something that changes since now every function resets the counter is that every first field is 0 in every class which is most of the time indistinguishable from default initialization. This can be changed by changing `count()` to `count(1)` or whatever other number.

Reply:
Good point. I have added a new constant and made all `count`s start from there.